packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -236,6 +236,15 @@ <h2>Then click:</h2>
       </br>
         <input title="create" type="submit" value="create" />
       </br>
+      <p>
+        <label for="setMeetingQualityLevel">Set Meeting Quality Level</label>
```

Comment:
Would prefer this to show that it is only being used at join:

```suggestion
        <label for="setMeetingQualityLevel">Meeting Join Quality Level</label>
```

Reply:
Does the rest of the sample need to reflect that meetingQuality can be either a string or an object? And what's the one we'd go with in the Sample?